### PR TITLE
ENG-4068 Add MfeDownloadManager tests

### PR DESCRIPTION
**Pull request by raxkaynan**

### CODE REVIEW

**Review by ichalagashvili — APPROVED** [review]

File: test/app-init/MfeDownloadManager.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    expect(screen.getByTestId('spinner')).toBeInTheDocument();
+  });
+
+  it('shows startup wait screen when polling', async () => {
+    fetchMfeConfigList.mockImplementationOnce(() => () => new Promise(resolve => resolve({})));
+
+    renderWithState(<MfeDownloadManager>test</MfeDownloadManager>);
+
+    expect(await screen.findByTestId('startup-wait-screen')).toBeInTheDocument();
+  });
```
Comment (Dv92) [inline test/app-init/MfeDownloadManager.test.js:28]: I would add another test to check the same behavior when the result is `null` or `undefined` (it should happen when the API has a status code of 500).

Comment (raxkaynan) [reply test/app-init/MfeDownloadManager.test.js:28]: It seems that response won't ever be `null` or `undefined` since it comes from `fetch`. Also, when a `5xx` error occurs, the Promise gets rejected. So I just added a test for that scenario.

**Review by Dv92 — APPROVED** [review]
